mne/viz/topomap.py

```diff
@@ -825,7 +825,7 @@ def plot_ica_components(ica, picks=None, ch_type=None, res=64,
     if merge_grads:
         from ..channels.layout import _merge_grad_data
     for ii, data_, ax in zip(picks, data, axes):
-        ax.set_title('IC #%03d' % ii, fontsize=12)
+        ax.set_title('IC #%03d' % (ii + 1), fontsize=12)
```

Comment:
in this case if you write # than it points to the index of the component so no +1

makes sense?